.github/workflows/lighthouserc.json

```diff
@@ -2,10 +2,16 @@
   "ci": {
     "collect": {
       "staticDistDir": "./audit/_build/",
-      "autodiscoverUrlBlocklist": [
-        "/genindex.html",
-        "/search.html",
-        "/_static/webpack-macros.html"
+      "url": [
+        "/admonitions.html",
+        "/api.html",
+        "/blocks.html",
+        "/generic.html",
+        "/lists.html",
```

Comment:
images.html is explicitly excluded because the performance score is likely the result of Picsum taking too long to load images

Reply:
This a is a good observation, and it is likely this is the root cause.